File: front/kaemeweb/src/app/modules/shared/models/PedidoModels/pedido.ts
export interface Pedido {
  num_Pedido?: number | null;
  cliente_Id: number;
  usuario_Id: number;
  data_Pedido: Date;
  ativo: boolean;
  cancelado: boolean;
  status_Pedido_Id: number;

  // Apenas para visualização
  cliente?: string;
  usuario?: string;
  data_Nasc?: Date;
}

File: back/KaemeWebApi/Dominio/Models/PedidoModels/Pedido.cs
namespace Dominio.Models
{
    public partial class Pedido
    {
        public int? Num_Pedido { get; set; }
        public int Cliente_Id { get; set; }
        public int Usuario_Id { get; set; }
        public DateTime Data_Pedido { get; set; }
        public bool Ativo { get; set; }
        public bool Cancelado { get; set; }
        public int Status_Pedido_Id { get; set; }

        // Apenas para visualização
        public string? Cliente { get; set; }
        public string? Usuario { get; set; }
        public DateTime? Data_Nasc { get; set; }

    }
}